### DISCUSSION TITLE
Server connection failed when i ask for image AUTOMATIC1111

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#2064] [Q&A]
When I configure the  AUTOMATIC1111 url with http://127.0.0.1:7860 I can choose the model and all is good when I valid the setting. But when i ask image with prompt i have the error : Server connection failed .

**Comment by justinh-rahb** [comment]
If you've installed with the standard `docker run` command or something resembling it, your base URL should be `http://host.docker.internal:7860/`

**Reply by ciscoivan** [reply]
I also faced the same problem, how should we issue the image query model correctly
